docs/source/pipelines.rst

```diff
@@ -4,194 +4,321 @@
+  are made.
+
+- **Production-ready structure**: The modular, git-integrated :ref:`pipeline structure
+  <pipeline-templates-key-concept>` dramatically simplifies the handoff from development to
+  production by ensuring that all model code, data, and configurations are easily reviewable and
+  deployable by ML engineers.
+
+Quickstarts
+-----------
+
+Prerequisites
+~~~~~~~~~~~~~
+
+.. _pipelines-installation:
+
 MLflow Pipelines is available as an extension of the
 `MLflow Python library <https://pypi.org/project/mlflow/>`_. You can install MLflow Pipelines
-from PyPI as follows:
+as follows:
 
-.. code-block:: sh
+- **Windows and Unix systems**: Install MLflow Pipelines from PyPI:
 
-    pip install mlflow[pipelines]  # for pip
+  .. code-block:: sh
 
-.. note::
-  MLflow Pipelines requires Make, which may not be preinstalled on some systems (e.g. Windows).
-  Please ensure Make is installed before using MLflow Pipelines.
+      pip install mlflow[pipelines]
 
-.. note::
-  You can install MLflow Pipelines from a Databricks Notebook by running
-  ``%pip install mlflow[pipelines]`` or install MLflow Pipelines on a Databricks Cluster
-  by following the instructions at
-  https://docs.databricks.com/libraries/cluster-libraries.html#install-a-library-on-a-cluster.
+  Note that MLflow Pipelines requires Make, which may not be preinstalled on some Windows systems.
+  Windows users must install Make before using MLflow Pipelines.
 
-  Note that `Databricks Runtime <https://docs.databricks.com/runtime/dbr.html>`_ version 11.0
+- **Databricks**: Install MLflow Pipelines from a Databricks Notebook by running
+  ``pip install mlflow[pipelines]``, or install MLflow Pipelines on a Databricks Cluster by
+  following the PyPI library installation instructions at
+  https://docs.databricks.com/libraries/cluster-libraries.html#install-a-library-on-a-cluster
+  and specifying the ``mlflow[pipelines]`` package string. Note that
+  `Databricks Runtime <https://docs.databricks.com/runtime/dbr.html>`_ version 11.0
   or greater is required in order to install MLflow Pipelines on Databricks.
 
-
-Key Concepts
+NYC taxi fare prediction example
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The `NYC taxi fare prediction example <https://github.com/mlflow/mlp-regression-example>`_
+uses the |MLflow Pipelines Regression Template| to develop and score accurate models on the
+`NYC Taxi (TLC) Trip Record Dataset
+<https://www1.nyc.gov/site/tlc/about/tlc-trip-record-data.page>`_. You can run the example by
+:ref:`installing MLflow Pipelines <pipelines-installation>` and running the `Jupyter example
+notebook <https://github.com/mlflow/mlp-regression-example/blob/master/notebooks/jupyter.ipynb>`_
+or the `Databricks example notebook
+<https://github.com/mlflow/mlp-regression-example/blob/master/notebooks/databricks.py>`_. For more
+information about importing and running a Databricks notebook, see
+https://docs.databricks.com/notebooks/notebooks-manage.html#import-a-notebook.
```

Comment:
We should provide the repos instructions here (https://docs.databricks.com/repos/sync-remote-repo.html). If a user just import the notebook, it won't work.

Reply:
Fixed - sorry about that :)